closed/OpenJ9.gmk

```diff
@@ -98,8 +98,8 @@ ifeq ($(OPENJDK_TARGET_OS),windows)
   # wrap PATH in quotes as it contains spaces (unix path)
   # INCLUDE, LIB are already wrapped in quotes (windows paths)
   EXPORT_MSVS_ENV_VARS := PATH="$(PATH)" INCLUDE=$(INCLUDE) LIB=$(LIB)
-  # disable mingw on windows, use default c compiler
-  EXPORT_NO_USE_MINGW := NO_USE_MINGW=true
+  # enable mingw on windows.  Note that the test is "ifdef", so don't define it at all.
+  # EXPORT_NO_USE_MINGW := NO_USE_MINGW=true
```

Comment:
Note its NO_USE_MINGW that shouldn't be defined, not EXPORT_NO_USE_MINGW

Reply:
EXPORT_NO_USE_MINGW is evaluated to set NO_USE_MINGW=true.
A cleaner way would be:
  EXPORT_NO_USE_MINGW :=<eol>

Reply:
Another alternative is to remove all references to EXPORT_NO_USE_MINGW from the file, since doing builds without mingw won't be a supported configuration. Disabling it was only temporary until mingw was working.